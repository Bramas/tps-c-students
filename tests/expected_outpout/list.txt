Command list

id:         263456
firt name:  Bob
last name:  Doe
birth year: 1900

id:         6452
firt name:  Bob
last name:  Doe
birth year: 2000

id:         15
firt name:  John
last name:  smith
birth year: 1979
