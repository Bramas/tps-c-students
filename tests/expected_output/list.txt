Command list

id:         263456
first name: Bob
last name:  Doe
birth year: 1900

id:         6452
first name: Bob
last name:  Doe
birth year: 2000

id:         15
first name: John
last name:  smith
birth year: 1979
